Makefile

```diff
@@ -141,6 +153,73 @@ liccheck:
 copyright-check:
 	python scripts/check_copyright.py
 
+####################
+### Clean and init commands
+####################
+
+.PHONY: clean
+clean: clean-build clean-pyc clean-test clean-docs
+
+.PHONY: clean-build
+clean-build:
+	rm -fr build/
+	rm -fr dist/
+	rm -fr .eggs/
+	rm -fr pip-wheel-metadata
+	find . -name '*.egg-info' -exec rm -fr {} +
+	find . -name '*.egg' -exec rm -fr {} +
+
+.PHONY: clean-docs
+clean-docs:
+#   Update when docs PR is merged to avoid conflicts
+# 	rm -fr docs/build/
+
+# This can be replaced with rmcache
```

Comment:
What do you mean?

Reply:
Sorry. Had another more unified one-line command. I've removed it so will remove this comment too.